Gemfile

```diff
@@ -17,6 +17,7 @@ group :development, :test do
   gem 'byebug',  '9.0.6', platform: :mri
   gem 'pry'
   gem 'pry-byebug'
+  gem 'rubocop', '~> 0.56.0', require: false
```

Comment:
Adding rubocop for RubyMine style checks. See https://www.jetbrains.com/help/ruby/robocop.html for setup instructions.